akka-actor-tests/src/test/java/akka/event/LoggingAdapterTest.java

```diff
@@ -128,7 +128,7 @@ protected Void match(Object event) {
                     LogEvent log = (LogEvent) event;
                     assertEquals(message, log.message());
                     assertEquals(level, log.level());
-                    assertEquals(mdc, log.mdc().toString());
+                    // assertEquals(mdc, log.mdc().toString());
                     if(cause != null) {
```

Comment:
ouch, this was not supposed to be here, please disregard (rebasing onto master will remove the need for this IDE workaround)